src/core/xref.js

```diff
@@ -439,9 +440,8 @@ function addToReferences(elem, cite, normative, term, conf) {
     return;
   }
 
-  const msg = `Normative reference to "${term}" found but term is defined "informatively" in "${cite}".`;
-  const title = "Normative reference to non-normative term.";
-  showWarning(msg, name, { title, elements: [elem] });
+  // This is used by the informative-dfn linter
+  informativeRefsInNormative.push({ term, cite, element: elem });
```

Comment:
Nit: Maybe call it "spec"?
```suggestion
  informativeRefsInNormative.push({ term, spec: cite, element: elem });
```